### PR TITLE
enh(multi): monitoring servers addition, group membership, network flows

### PR DESCRIPTION
**Pull request by cgagnaire**
## Description

Update communication type notes when adding monitoring servers (SSH with Remote Server deprecated, SSH legacy port not used anymore).
Update centreon-gorgone group membership.
Update network flows tables (remove HTTP between Central and Remote Server, better show difference between Poller and Remote Server, add registration action).

## Target serie

- [ ] 20.04.x
- [x] 20.10.x (master)

### CODE REVIEW

File: en/monitoring/monitoring-servers/add-a-poller-to-configuration.md [hunk 1]
```diff
@@ -63,6 +63,11 @@ recommended).
 
 Click on **Save**.
 
+> Note that the *SSH Legacy port* is not used anymore and will be removed.
```
Comment (lpinsivy) [inline en/monitoring/monitoring-servers/add-a-poller-to-configuration.md:66]: This notice should be in the "Using SSH" tab

File: en/monitoring/monitoring-servers/add-a-poller-to-configuration.md [hunk 2]
```diff
@@ -159,6 +158,11 @@ connection protocol**. Define the suitable **port**.
...
```
Comment (lpinsivy) [inline en/monitoring/monitoring-servers/add-a-poller-to-configuration.md:161]: All SSH part must be in "Using SSH" tab

File: fr/monitoring/monitoring-servers/add-a-poller-to-configuration.md [hunk 3]
```diff
@@ -58,15 +58,15 @@ Editer la configuration du Poller fraichement créé, et sélectionner **ZMQ**
...
-> Il est à noter qu'un port SSH est toujours necessaire (SSH Legacy port) pour
-> l'extension de découverte de service (Service Discovery).
->
-> Cette extension utilisera bientôt le protocole de communication de Gorgone.
-
 ![image](../../assets/monitoring/monitoring-servers/poller-edit-zmq.png)
 
 Cliquer sur **Sauvegarder**.
 
+> Notez que le *SSH Legacy port* n'est plus utilisé et sera supprimé.
```
Comment (lpinsivy) [inline fr/monitoring/monitoring-servers/add-a-poller-to-configuration.md:65]: All SSH part must be in "Using SSH" tab

**Review by lpinsivy — CHANGES_REQUESTED** [review]

File: en/monitoring/monitoring-servers/add-a-poller-to-configuration.md (hunk 1 above)
Comment (cgagnaire) [reply en/monitoring/monitoring-servers/add-a-poller-to-configuration.md:66]: This legacy port is not the Gorgone port if Gorgone is in SSH mode.
It was the one used by Centcore and Service Discovery, and is not used anymore, except maybe by custom user scripts.

**Review by lpinsivy — APPROVED** [review]

**Pull request merged**